hondana/http.py

```diff
@@ -473,9 +528,7 @@ async def request(self, route: Union[Route, DownloadRoute], **kwargs: Any) -> An
                         return data
 
                     if response.status == 429:
-                        assert retry is not None
-                        delta = retry - datetime.datetime.now()
-                        sleep = delta.total_seconds() + 1
+                        sleep = float(response.headers["retry-after"])
```

Comment:
The `x-ratelimit-retry-after` header is a UNIX timestamp (from MangaDex anyway).
Unless I'm mistaken, this will just sleep for 163 million seconds.

Reply:
`retry-after` is the retry-time in seconds, it's given on any 429
`x-ratelimit-retry-after` is the ratelimit reset time as a unix timestamp